lobot/lobot.js

```diff
@@ -22,14 +31,14 @@ async function LoadResource(name, type) {
     : null;
 }
 
-async function Serve(resource) {
+async function Serve(resource, config) {
   const name = resource.split(`:`)[0];
   const type = resource.split(`:`)[1];
   if (type === `css` || type === `js` || type === `txt`) {
     return await LoadResource(name, type);
   }
   if (type === `html`) {
-    return await fragmentBundle[name];
+    return fragmentBundle[name];
```

Comment:
Html modules are invoked within the template as such we don't need to wait here since we are not invoking and triggering the nested lobot resource functions.